### ISSUE TITLE
New HTML file

### ISSUE THREAD

**Issue by rajput999** [DanishAbdullahPy/Html-for-freshmans#2]
Can we add any html file?

**Comment by DanishAbdullahPy** [comment]
Hacktoberfest contribution will count from 1oct so wait pateintly and yes
you can contribute anything if you are beginner

**Comment by SRUJANA2199** [comment]
hi, i would like to work on this can you assign me this ?

**Comment by DanishAbdullahPy** [comment]
Yes, I can do it.

**Comment by DanishAbdullahPy** [comment]
wait till 3 ocotber i will assign

**Comment by rajput999** [comment]
ok, thanks

**Comment by om453** [comment]
hello i want to contribute as a beginner pls assign me this

**Comment by DanishAbdullahPy** [comment]
Yes, of course you can. But dont just write a boiler plate of HTML

**Comment by njangid22** [comment]
I want to contribute as a beginner please assign me this.

**Comment by VoraAnshul18** [comment]
please Assign me , I will do it

**Comment by AparajitaSingh121416** [comment]
hey @DanishAbdullahPy , i want to contribute to hacktoberfest, and i am a beginner so please assign this issue to me.

**Comment by MayankChoudhary21** [comment]
Please assign me this issue

**Comment by SUMITHSAI** [comment]
Please accept my PR, that contains my html file and css inline file.
